cmd/kyma/alpha/create/scaffold/default_cr.go

```diff
@@ -0,0 +1,132 @@
+package scaffold
+
+import (
+	"fmt"
+	"gopkg.in/yaml.v3"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+)
+
+type CustomResourceDefinition struct {
+	Spec struct {
+		Group string `yaml:"group"`
+		Names struct {
+			Kind         string `yaml:"kind"`
+			SingularKind string `yaml:"singular"`
+		} `yaml:"names"`
+		Versions []struct {
+			Name   string `yaml:"name"`
+			Schema struct {
+				OpenAPIV3Schema struct {
+					Properties map[string]Property `yaml:"properties"`
+				} `yaml:"openAPIV3Schema"`
+			} `yaml:"schema"`
+		} `yaml:"versions"`
+	} `yaml:"spec"`
+}
+
+type Property struct {
+	Type       string              `yaml:"type"`
+	Properties map[string]Property `yaml:"properties"`
+}
+
+func (cmd *command) generateDefaultCR() error {
```

Comment:
All the generator funcs are good candidates for unit tests, WDYT?

Reply:
In my opinion, the e2e test should be sufficient as per the AC. It was also discussed with @ruanxin that it is a good approach to generate the files and run the create a module from them.

Since the generator files are very dynamic (eg. will change when. comments for fields are changed, new fields are introduced, etc) and each generated file is not necessarily wrong, therefore, I would refrain from unit testing the generators. Furthermore, for manifest, sec-scanners-config, and module-config, we would be only testing the yaml generation from an external library. It would only make sense for default cr generation because we have some unmarshalling, and unique parsing logic. But the E2e test would already check if the default cr exists (got generated properly) and if it is valid so. So do you think unit tests are necessary?